config/vitest.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from "vitest/config";
+
+export default defineConfig({
+  test: {
+    include: ["**/*.spec.js"],
```

Comment:
```suggestion
    include: ["packages/alfa-*/test/**/*.spec.js"],
```
I prefer being less "catch all" when possible 😃

Reply:
This gives some problems with the `--dir packages/alfa-xxx` command line option. It seems the include path is relative to the directory being passed in the option.

I have looked at the [workspace](https://vitest.dev/guide/workspace.html) feature. It can limits us to the `alfa-*` packages and enables testing individual packages. It will also prefix the package names in the tests which I find also a nice little addition.

The only thing annoying is that it takes the package name from its `package.json` meaning we have to write `yarn test --project (at)siteimprove/alfa-xxx` 😕

Reply:
Yeah, we might have to do `yarn workspace @siteimprove/alfa-foo test` or something similar indeed. But maybe we can salvage the simpler command (+ shell completion by path, which is the default behaviour and thus works out of the box!) by doing some magic around the `test` script in `package.json` (we already did so in `scripts/test.mjs`…)

Some magic like `test: vitest --project $(sed 's+packages+@siteimprove+)` (plus some more magic around no paramtere 🙈), or just keep a simple `scripts/test.mjs` that wraps vitest with the correct config based on its parameters?


Ultimately, and due to https://github.com/Siteimprove/alfa-integrations/pull/102#issuecomment-2441608723 we might want each individual package to have its own `test` script (and call vitest, probaly with shared config if possible), and the top-level script will be `yarn workspaces --all foreach test` (I also like the package prefixing). Then we need a wrapper to do `yarn workspace @siteimprove/alfa-foo test` nicely (and with directory based completion, unless a nice `yarn workspace` completion utility exists…)

Reply:
Just realized there might be some confusing naming overlap here 😅 The workspace feature I looked at is actually a vitest thing (I looks like it tries to do something similar to yarn workspaces). It might also make sense to try and use yarn workspaces, but what I'm looking at currently involves only vitest. Sorry that wasn't very clear.

I'll push some changes soon, then I hope it should be more clear what it's about.